tests/system/test_system.py

```diff
@@ -691,9 +889,14 @@ def test_list_backups(self):
         instance = Config.INSTANCE
         expire_time_1 = datetime.utcnow() + timedelta(days=21)
         expire_time_1 = expire_time_1.replace(tzinfo=UTC)
+        version_time_1 = datetime.utcnow() - timedelta(minutes=5)
```

Comment:
There is an assumption here that the instance is 5 minutes old. I think this is a fair assumption, but just want to make sure that we know the database that we are backing up is always at least 5 minutes old when we reach this, otherwise we can get flaky tests.